lib/services/event_service.ex

```diff
@@ -82,6 +82,27 @@ defmodule Aprb.Service.EventService do
                         }]",
           unfurl_links: true
          }
+      "conversations" ->
+        if event["properties"]["buyer_outcome"] == "other" do
+          %{
+            text: ":phone: #{event["object"]["display"]} responded on https://radiation.artsy.net/accounts/2/conversations/#{event["properties"]["radiation_conversation_id"]}",
```

Comment:
Seems like `radiation_account_id` could be a property of the event, or--medium term--exposing a view of the conversation in Impulse makes the most sense.